src/utils/OnboardingProvider.tsx

```diff
@@ -50,7 +52,8 @@ export default function OnboardingProvider({
 }: {
   children: ReactNode;
 }) {
-  const profileCtx = useContext(ProfileContext);
+  const auth = useAuth();
+  const profileCtx = useProfile();
```

Comment:
microscopic nit: should we standardize to call it `profile` vs `profileCtx`? not sure how much this matters. fine with either. 

side tip that im sure you know already: if we choose to refactor, you can put the text cursor on `profileCtx` then hit F2 when the keyword is highlighted to rename all instances of `profileCtx`

Reply:
nit: I also prefer `profile` vs `profileCtx` lol

Reply:
i think the reason this is `profileCtx` here is because there is also this `useState` defined a few lines later:

`const [profile, setProfile] = useState<Partial<Profile>>({});`

shall i rename this instead? and if so, what to? maybe `onboardingProfile` or `profileObj` or `profileState` or smth?

Reply:
oh mb i overlooked that

maybe we can call it `userProfile`?

Reply:
hmmm `userProfile` (and `userState`, my other idea) also don't work because there are types imported from `@/types/schema` that have the same name. imma use `profileObj` lol

Reply:
Can you elaborate on where they're imported? I don't see `userProfile` or `userState` being defined (nor exported) from `@/types/schema`
![image](https://github.com/calblueprint/immigration-justice-project/assets/67077248/b0681ccf-5949-4fff-8223-31a8072ee43b)
![image](https://github.com/calblueprint/immigration-justice-project/assets/67077248/d5c12314-80e5-459e-9bd0-260c26b8430a)

Reply:
but `profileObj` would work too

Reply:
i actually. have literally no clue what i was referring to LMAO. `userProfile` worked! replacing